python/cudf_polars/cudf_polars/experimental/statistics.py

```diff
@@ -574,7 +580,11 @@ def copy_child_unique_counts(column_stats_mapping: dict[str, ColumnStats]) -> No
 
 
 @update_column_stats.register(IR)
-def _(ir: IR, stats: StatsCollector, config_options: ConfigOptions) -> None:
+def _(
```

Comment:
These formatting changes can be reverted.

Reply:
They can be reverted